main.go

```diff
@@ -99,11 +99,17 @@ func launch(c *cli.Context) {
 	}(resultChan)
 
 	go func(rc chan error) {
-		err := hostlabels.StartHostLabelSync(c.Int("host-labels-update-interval"), kClient)
+		err := hostwatch.StartHostLabelSync(c.Int("host-labels-update-interval"), kClient)
```

Comment:
Do you think it would make sense to merge hostwatch. StartHostLabelSync and hostwatch. StartHostStatusSync into one call? Something like hostWatch.startSync. Then inside startSync, you can split into 2